integration/runner/runner.go

```diff
@@ -44,7 +44,7 @@ const (
 	testTimeout    = 15 * time.Minute
 )
 
-var cadvisorTimeout = flag.Duration("cadvisor_timeout", 15*time.Second, "Time to wait for cAdvisor to come up on the remote host")
+var cadvisorTimeout = flag.Duration("cadvisor_timeout", 30*time.Second, "Time to wait for cAdvisor to come up on the remote host")
```

Comment:
I still a bit concern. Have you tried without containerd running? How long does it take to try connecting containerd?

Reply:
I am going to get back on this. Running on a single machine doesnt seem to be an issue . Its only when the CI runs ssh and runs it that the timeout needs to be increased. Will come back with a valid explaination